src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -202,15 +236,28 @@
                                             @foreach (var subItem in item.Navigations)
                                             {
                                                 <li>
-                                                    <a href="@subItem.Link"
-                                                       class="dropdown-item"
-                                                       aria-label="@subItem.NavigationText.AriaLabel">
-                                                        @if (!string.IsNullOrEmpty(subItem.Icon))
-                                                        {
+                                                    @if (string.IsNullOrEmpty(subItem.NavigationText.Label)
+                                                        && !string.IsNullOrEmpty(subItem.Icon)
+                                                        && !string.IsNullOrEmpty(subItem.NavigationText.Title))
+                                                    {
+                                                        <a href="@subItem.Link"
+                                                           class="dropdown-item"
+                                                           title="@item.NavigationText.Title">
                                                             <span class="@subItem.Icon"></span>
-                                                        }
-                                                        @subItem.NavigationText.Label
-                                                    </a>
+                                                            @subItem.NavigationText.Label
```

Comment:
The label text here can be removed.

Reply:
Resolved in commit 9b9fd46116bc0d3399fe1289b3f3852d4fd5576c